### PR TITLE
chore: rename document components

### PR DESCRIPTION
**Pull request by echl**

### CODE REVIEW

File: packages/client/hmi-client/src/page/Home.vue [hunk 1]
```diff
@@ -81,7 +81,7 @@
 						</div>
 						<ul>
 							<li v-for="(document, j) in project.relatedDocuments" :key="j">
-								<DocumentCard :document="document" @click="selectDocument(document)" />
+								<TeraDocumentCard :document="document" @click="selectDocument(document)" />
```
Comment (Tom-Szendrey) [inline packages/client/hmi-client/src/page/Home.vue:84]: We are using Kebab Case for this 
<tera-document-card..

File: packages/client/hmi-client/src/page/Home.vue [hunk 2]
```diff
@@ -93,7 +93,7 @@
 					<div class="carousel">
 						<ul>
 							<li v-for="i in [0, 1, 2, 3, 4, 5]" :key="i">
-								<DocumentCard />
+								<TeraDocumentCard />
```
Comment (Tom-Szendrey) [inline packages/client/hmi-client/src/page/Home.vue:96]: Using kebab case for this
<tera-document-card..

File: packages/client/hmi-client/src/page/Home.vue [hunk 3]
```diff
@@ -170,15 +170,15 @@
...
+import TeraSelectedDocumentPane from '@/components/documents/tera-selected-document-pane.vue';
 import { IProject } from '@/types/Project';
 import { XDDSearchParams } from '@/types/XDD';
 import { DocumentType } from '@/types/Document';
 import { searchXDDDocuments } from '@/services/data';
 import useResourcesStore from '@/stores/resources';
 import useQueryStore from '@/stores/query';
 import ProjectCard from '@/components/projects/ProjectCard.vue';
-import DocumentCard from '@/components/documents/DocumentCard.vue';
+import TeraDocumentCard from '@/components/home/tera-document-card.vue';
```
Comment (YohannParis) [inline packages/client/hmi-client/src/page/Home.vue:181]: Why should this component be moved, we might use those cards elsewhere

Comment (echl) [reply packages/client/hmi-client/src/page/Home.vue:181]: We don't use this component elsewhere though, it is quite specific to the home page.

**Review by YohannParis — APPROVED** [review]

**Pull request merged**